src/django_clickhouse/migration_operators.py

```diff
@@ -0,0 +1,48 @@
+from infi import clickhouse_orm
+from .configuration import config
+
+
+class HintParamMixin:
+    def __init__(self, *args, hints=None, **kwargs):
+        if hints is None:
+            hints = dict()
+        self.hints = hints
+        super(HintParamMixin, self).__init__(*args, **kwargs)
+        if not any([
+            isinstance(self, clickhouse_orm.ModelOperation),
+            'model' not in hints.keys(),
+            'force_migrate_on_databases' not in hints.keys()
+        ]):
+            self.hints['force_migrate_on_databases'] = (config.DEFAULT_DB_ALIAS,)
```

Comment:
Why `default` database is used when hints are absent? This is wrong behavior: when hints are not present router should decide what databases operation should be applied to. Here you state that all operations are applied to `default` database if no hints are given directly, ignoring `ClickHouseModel.migrate_replicated_db_aliases`, `ClickHouseModel.migrate_non_replicated_db_aliases` attributes or any behavior a library user can write using custom router.

Reply:
the default router choose databases based on model **migrate_replicated_db_aliases** and **migrate_non_replicated_db_aliases** in model operations which is not define in non model operations.
same as model operations which use default database when user did not override **migrate_replicated_db_aliases**, **migrate_non_replicated_db_aliases** in model class, I decide to use default database in absence of hints only when operation not instance of model operation (which is prevent ignoring **migrate_replicated_db_aliases**, **migrate_non_replicated_db_aliases**).